lib/create-options.js

```diff
@@ -10,6 +11,9 @@ function createOptions(cwd, context) {
   options.env['npm_loglevel'] = 'error';
   options.env['npm_config_tmp'] = context.npmConfigTmp;
 
+  _.forEach(context.module.envVar, function (val, key) {
```

Comment:
you still need to do a check for both module and envVar here

if ( context.module && context.module.envVar)